javaslang/src/main/java/javaslang/collection/HashSet.java

```diff
@@ -559,7 +559,7 @@ public boolean contains(T element) {
             return empty();
         } else {
             final HashArrayMappedTrie<U, U> that = foldLeft(HashArrayMappedTrie.empty(),
-                    (tree, t) -> addAll(tree, mapper.apply(t)));
+                                                            (tree, t) -> addAll(tree, mapper.apply(t)));
```

Comment:
Please don't reformat code. Do you use IntelliJ IDEA? Then please use our setting `<javaslang-root>/.ide/idea-settings.jar`